torch/linalg/__init__.py

```diff
@@ -339,6 +339,88 @@
     (tensor(3.7417), tensor(11.2250))
 """)
 
+cond = _add_docstr(_linalg.linalg_cond, r"""
+linalg.cond(input, p=None, *, out=None) -> Tensor
+
+Computes the condition number of a matrix :attr:`input`,
+or of each matrix in a batched :attr:`input`, using the matrix norm defined by :attr:`p`.
+The condition number is defined as the matrix norm of
+:attr:`input` times the matrix norm of the inverse of :attr:`input`.
+
+This function supports ``float``, ``double``, and only on CPU, ``cfloat`` and ``cdouble`` dtypes for :attr:`input`.
+
+.. note:: For `p = {None, 2, -2}` there is a relation between singular value decomposition and matrix norm.
```

Comment:
Suggest simplifying this to: 

"For \`p = {None, 2, -2}\` the condition number is computed as the ratio between the largest and smallest singular values computed using  :func:\`torch.linalg.svd\`. For these norms :attr:\`input\` may be a non-square matrix or batch of non-square matrices. For other norms, however, :attr:\`input\` must be a square matrix or a batch of square matrices, and if this requirement is not satisfied a RuntimeError will be thrown."

Reply:
Your variant is nicer to read, thanks!